python/ray/data/preprocessors/chain.py

```diff
@@ -4,6 +4,7 @@
 from ray.util.annotations import PublicAPI
 
 if TYPE_CHECKING:
+    from ray.air.util.data_batch_conversion import BatchFormat, BlockFormat
```

Comment:
do we need to stick this in TYPE_CHECKING? The base `preprocessor.py` is able to import this directly.

Reply:
Sure!